### PR TITLE
Removing .spi from generated package names

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: google-cloud-vision/src/main/java/com/google/cloud/vision/v1/PagedResponseWrappers.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.cloud.language.spi.v1;
```
Comment (jabubake) [inline google-cloud-vision/src/main/java/com/google/cloud/vision/v1/PagedResponseWrappers.java]: expected : language -> vision ?

File: google-cloud-language/src/main/java/com/google/cloud/language/v1/PagedResponseWrappers.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.cloud.vision.spi.v1;
```
Comment (jabubake) [inline google-cloud-language/src/main/java/com/google/cloud/language/v1/PagedResponseWrappers.java]: Is this expected : vision -> language ?

Comment (jabubake) [inline google-cloud-language/src/main/java/com/google/cloud/language/v1/PagedResponseWrappers.java]: Ignore, I just noticed the change in path.

Comment (garrettjonesgoogle) [inline google-cloud-language/src/main/java/com/google/cloud/language/v1/PagedResponseWrappers.java]: This is git getting confused. Notice that the destination file is language here too. If you browse the final state of the code after my change, you'll see it's in order.

File: google-cloud-vision/src/main/java/com/google/cloud/vision/v1/PagedResponseWrappers.java (hunk 1 above)
Comment (jabubake) [inline google-cloud-vision/src/main/java/com/google/cloud/vision/v1/PagedResponseWrappers.java]: Ignore, is there a reason for this file swap ?

**Pull request merged**